src/content/docs/guides/integrate-in-vcs.mdx

```diff
@@ -63,5 +65,14 @@ Biome doesn't check what's changed, this means that even adding spaces or newlin
 Alternatively, you can use the option `--since` to specify an arbitrary branch. This option **takes precedence** over the option `vcs.defaultBranch`. For example, you might want to check your changes against the `next` branch:
 
 ```shell
-biome format --changed --since=next
+biome check --changed --since=next
+```
+
+### Process only staged files
+
+Before committing your changes, you may want to check the formatting and lints files that have been added to the _index_, also known as _staged files_.
+Add the `--staged` option to your command, to process only those files:
+
+```shell
+biome check --staged
```

Comment:
We may want to address that this option isn't available in the biome ci command, and explain why